cypress/integration/step_definitions/backend/aa_uprade_downgrade_usersubscription.cy.js

```diff
@@ -0,0 +1,14 @@
+import {Then} from "cypress-cucumber-preprocessor/steps";
+
+
+
+ Then('User should be able to upgrade or downgrade usersubscription to {string}', (tier_type)=>{
+   cy.upGradeDownGradeUsersubscription(tier_type).then((Response)=>{ 
+    expect(Response.body).to.eq('Succesfully updated')
+   });     
```

Comment:
Does response return only info message as `Succesfully updated`? If not, please also assert other values. If this is the only value in response, maybe you can check if the subscription is updated in userSupscription endpoint. Also it would be great if you assert credits with the new plan, just to make sure we still have the remaining counts and the new counts.

Reply:
done